.github/workflows/build.yml

```diff
@@ -31,11 +31,49 @@ jobs:
       - uses: dsaltares/fetch-gh-release-asset@a40c8b4a0471f9ab81bdf73a010f74cc51476ad4 # v1.1.1
         with:
           repo: thebrowsercompany/firebase-cpp-sdk
-          version: tags/20230921.0
+          version: tags/20240306.0
```

Comment:
Seems unreleated?

Reply:
Hmm, actually, I wonder if we really need the newer firebase images or not 🤔. I think that I was seeing an error that made me think that I needed the latest snapshot for the C++ SDK that we are using. I'm happy to roll this back if you think that is better.